src/index.ts

```diff
@@ -168,6 +207,13 @@ export const plugin: Plugin = {
 									logger.debug(lineLength, printWidth);
 									tempToken = tokens[++tempIndex] as AttributeToken | EndAttributesToken;
 								}
+								if (enableSortAttributes) {
+									const startAttributesIndex: number = tokens.indexOf(token);
+									const endAttributesIndex: number = tempIndex;
+									tokens = partialSort(tokens, startAttributesIndex + 1, endAttributesIndex, (a, b) =>
```

Comment:
Only sort if the difference of `startAttributesIndex` to `endAttributesIndex` is greater then `1`